src/controllers/trigger/common/trigger.js

```diff
@@ -53,7 +51,7 @@ async function getSitesToAudit(dataAccess, url, deliveryType, orgs) {
     const site = await dataAccess.getSiteByBaseURL(url);
     sitesToAudit = site ? [site] : [];
   }
-  return sitesToAudit.filter((site) => !isAuditsDisabled(site, orgs[site.getOrganizationId()]));
+  return sitesToAudit;
```

Comment:
The js doc needs to be updated, because in the new version we don't have a flag to disable all handler types for a site.